pkg/scheduler/plugins/nodeorder/nodeorder.go

```diff
@@ -145,6 +148,9 @@ func calculateWeight(args framework.Arguments) priorityWeight {
 	// Checks whether imagelocality.weight is provided or not, if given, modifies the value in weight struct.
 	args.GetInt(&weight.imageLocalityWeight, ImageLocalityWeight)
 
+	// Checks whether selectorspread.weight is provided or not, if given, modifies the value in weight struct.
+	args.GetInt(&weight.selectorSpreadWeight, ImageLocalityWeight)
```

Comment:
why use ImageLocalityWeight in `args.GetInt(&weight.selectorSpreadWeight, ImageLocalityWeight)`

Reply:
a copy-paste mistake 😿 , fixed.